src/main/java/potaton/Potaton.java

```diff
@@ -0,0 +1,299 @@
+package potaton;
+
+import potaton.task.Deadline;
+import potaton.task.Events;
+import potaton.task.Task;
+import potaton.task.ToDos;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Potaton {
+
+    public static final String TASK_LIST = "list";
+    public static final String TASK_TODO = "todo";
+    public static final String TASK_EVENT = "event";
+    public static final String TASK_DEADLINE = "deadline";
+    public static final String TASK_MARK = "mark";
+    public static final String TASK_UNMARK = "unmark";
+    public static final String TASK_DELETE = "delete";
+    public static final String EXIT = "bye";
+
+    public static final String LOADTASK_TODO = "T";
+    public static final String LOADTASK_EVENT = "E";
+    public static final String LOADTASK_DEADLINE = "D";
+
+    public static final String INIT_FILE_PATH = "potaton.txt";
+    public static final String FILE_MISSING_MESSAGE = "File does not exist. Creating file.";
+    public static final String FILE_FOUND_MESSAGE = "File found. Loading file into database.";
+
+    public static void main(String[] args) throws IOException {
+        printWelcomeMessage();
+        ArrayList<Task> tasks = new ArrayList<>();
+        loadFileContent(tasks);
+        parseUserCommand(tasks);
+        saveContentToFile(tasks);
+        printBye();
+    }
+
+    private static void parseUserCommand(ArrayList<Task> tasks) {
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+        String command = line.split(" ")[0];
+        while (!command.equals(EXIT)) {
+            switch (command) {
+            case TASK_LIST:
+                printList(tasks);
+                break;
+            case TASK_UNMARK:
+                unmarkTask(line, tasks);
+                break;
+            case TASK_MARK:
+                markTask(line, tasks);
+                break;
+            case TASK_TODO:
+                try {
+                    addToDo(line, tasks);
+                } catch (IndexOutOfBoundsException e) {
+                    // Refactor this later
+                    System.out.println("_____________________________________________\n"
+                            + "Error: The description of a todo cannot be empty." + "\n"
+                            + "_____________________________________________\n"
+                    );
+                }
```

Comment:
This block of code could be refactored so as to ensure [SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard).